ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -73,7 +72,7 @@ const PostHeaderBanner = ({
       />
       <PostBreakingNewsLabel
         isBreakingNews={isBreakingNews}
-        breakingNewsLabelText={breakingNewsLabelText}
+        breakingNewsLabelText={translations.liveExperiencePage.breaking}
```

Comment:
Just wondering if we should put the fallback value here e.g.  {translations.liveExperiencePage.breaking ?? 'Breaking'}. Rather than using a default value for the prop (Line 27). I'm not sure the pros and cons on this, but I personally find it easier to understand.

Reply:
The service context shouldn't fail but if it did the entire app would likely not work. 

I think its a good idea to have fallbacks, just in the event we have to remove translations for whatever reason, it means that something will always be displayed.